feg/gateway/services/aaa/servicers/accounting.go

```diff
@@ -101,8 +101,9 @@ func (srv *accountingService) InterimUpdate(_ context.Context, ur *protos.Update
 	srv.sessions.SetTimeout(sid, srv.sessionTout, srv.timeoutSessionNotifier)
 
 	imsi := metrics.DecorateIMSI(s.GetCtx().GetImsi())
-	metrics.OctetsIn.WithLabelValues(s.GetCtx().GetApn(), imsi).Add(float64(ur.GetOctetsIn()))
-	metrics.OctetsOut.WithLabelValues(s.GetCtx().GetApn(), imsi).Add(float64(ur.GetOctetsOut()))
+	msisdn := s.GetCtx().GetMsisdn()
+	metrics.OctetsIn.WithLabelValues(s.GetCtx().GetApn(), imsi, msisdn).Add(float64(ur.GetOctetsIn()))
```

Comment:
I have never checked before this part of the code, do you know what GetOctetsIn and GetOctetsOut are?

Reply:
I respond to myself with a picture. It is in fact the amount of traffic reported by AP to the accounting service

![image](https://user-images.githubusercontent.com/16157139/90685910-c8b5f480-e26a-11ea-96d5-1a7582996e9e.png)